.github/actions/verify-token/action.yml

```diff
@@ -5,24 +5,25 @@ description: "Verify a token"
 inputs:
   slsa-workflow-recipient:
     description: "The audience of the token"
-    type: string
     required: true
 
   slsa-unverified-token:
     description: "The token to verify"
-    type: string
     required: true
 
   token:
     description: "The GitHub Actions token."
     required: false
-    default: ${{ github.token }}
 
   output-predicate:
     description: "The filename to place the output SLSA predicate based on the verified token"
-    type: string
     required: true
 
+  slsa-version:
+    description: "The version of SLSA provenance format to output in the predicate."
+    required: false
+    default: "1.0-rc1"
```

Comment:
thank you for the RC change - just FYI we will need to update https://github.com/slsa-framework/slsa-github-generator/blob/80da39e3f22bc13d965dfbccc408098ca2f362fe/.github/workflows/delegator_generic_slsa3.yml#L229

Reply:
I'll go ahead and make the change on this PR.